generic3g/specs/FrequencyAspect.F90

```diff
@@ -28,22 +30,18 @@ module mapl3g_FrequencyAspect
       procedure :: set_timestep
       procedure :: get_accumulation_type
       procedure :: set_accumulation_type
+      procedure :: get_reference_time
+      procedure :: set_reference_time
```

Comment:
Does this class need any of these accessors?  (Including the ones that precede this PR.)

The tests don't use the accessors.  And external code certainly does not.

Reply:
I added the accessors because the reference time is not in release/MAPLv3. I added it for this PR. I didn't add it to the constructor yet. Is reference time known at instantiation of the FrequencyAspect? If so and we don't need to change it after instantiation, then I can add it to the constructor and remove the setter. The getter is needed because the field for reference time is private. I don't know the use cases, so I don't know what is needed and/or appropriate.

I didn't want to break existing code by adding reference time to the constructor, and I needed to set and get reference time.

Reply:
Yes - refTime and timeStep are both known prior to constructing the varspec.   If we use the yaml mechanism all is good, but this likely needs work if we use the ACG, because we don't want the users to manually include these in their tables.  Burn that bridge when we cross it. (Favorite mixed metaphor.)

Still not sure I see why you need any getter's.   The FrequencyAspect makes the action and can pass refTime and timeStep to the action constructor without an accessor.